services/timers/vmp-mapping/vmp-mapping.sh

```diff
@@ -4,6 +4,8 @@ set -euo pipefail
 # this needs to be called DATABASE_URL in cohortextractor, but we do not want to expose it in telemetry
 export DATABASE_URL=$FULL_DATABASE_URL
 
+export OTEL_EXPORTER_OTLP_HEADERS="$(echo $OTEL_EXPORTER_OTLP_HEADERS | sed 's#%20# #g')"
```

Comment:
Could the "%20" characters be removed in the .env file (or wherever this is being set)? I'm not sure if it's needed elsewhere or not?

Reply:
Sadly not that simple: https://github.com/opensafely-core/backend-server/issues/178